website/pages/components/toggle-group/specifications.tsx

```diff
@@ -0,0 +1,21 @@
+import Head from "next/head";
+import type { ReactElement } from "react";
+import ToggleGroupPageLayout from "../../../screens/components/toggle-group/ToggleGroupPageLayout";
+import ToggleGroupSpecsPage from "../../../screens/components/toggle-group/specs/ToggleGroupSpecsPage";
+
+const Specifications = () => {
+  return (
+    <>
+      <Head>
+        <title>ToggleGroup Specs — Halstack Design System</title>
+      </Head>
```

Comment:
Same as above. I would change the title to `Toggle Group`, like for `Text Input`, `Number Input` and the rest of components with more than one word in its title.